packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -943,30 +970,47 @@ export default decorate([
                         </FormGroup>
                       )}
                       {state.isFull && (
+                        <div>
+                          <FormGroup>
+                            <label htmlFor={state.compressionId}>
+                              <strong>{_('compression')}</strong>
+                            </label>
+                            <SelectCompression
+                              id={state.compressionId}
+                              onChange={effects.setCompression}
+                              value={compression}
+                            />
+                          </FormGroup>
+                          <FormGroup>
+                            <label>
+                              <strong>{_('offlineBackup')}</strong>{' '}
+                              <Tooltip content={_('offlineBackupInfo')}>
+                                <Icon icon='info' />
+                              </Tooltip>{' '}
+                              <input
+                                checked={offlineBackup}
+                                onChange={effects.setOfflineBackup}
+                                type='checkbox'
+                              />
+                            </label>
```

Comment:
We must warn user that all VMs will be shutdown at the same time.

Reply:
It's done in the tooltip.